### ISSUE TITLE
TPOT regression with separate test and train dataset

### ISSUE THREAD

**Issue by sonuthomas1996** [EpistasisLab/tpot#839]
Hi, I'm trying out tpot for time series problem and i have two separate dataset, one for training and one for testing. While using normal train test split it gives you x_train, y_train, x_test, y_test. In this case im having separate dataset. Since i'm new to machine learning i'm confused then what are the parameters i need to pass for fit function , normally x_train and y_train is passed. In my case what should i pass? If someone could help me perform the timeseries it would be very helpful. My dataset contains 14 columns , one is the date and time and other is output variable and the related features.

_Originally posted by @sonuthomas1996 in https://github.com/EpistasisLab/tpot/issues/345#issuecomment-468139420_

**Comment by weixuanfu** [comment]
I think you may manually split dataset into train/test set instead of using 'train_test_split'. (see [this link](https://stackoverflow.com/questions/50879915/how-to-split-data-using-time-based-in-test-and-train-respectively))

Also for using TPOT, you may try `cv=TimeSeriesSplit(n_splits=5)` for time series problem. Please see the [TimeSeriesSplit](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.TimeSeriesSplit.html) for more details.
